src/main/java/com/iota/iri/utils/Converter.java

```diff
@@ -232,9 +231,9 @@ public static String asciiToTrytes(String input) {
         StringBuilder sb = new StringBuilder(80);
         for (int i = 0; i < input.length(); i++) {
             int asciiValue = input.charAt(i);
-            // If not recognizable ASCII character, return null
             if (asciiValue > 255) {
-                return null;
+                throw new IllegalArgumentException("The ascii trytes String contains un illegal tryte character:" +
+                    " char= " + ((char)asciiValue) + ", (" + asciiValue +")");
             }
```

Comment:
returning null will only cause a null pointer exception - and a hard to find one! Either throw a null pointer exception here , or else a more useful message such as this suggested message.